### ISSUE TITLE
E482 can't create file

### ISSUE THREAD

**Issue by johatfie** [yegappan/mru#18]
I have the following line in my .vimrc file:

```
    let MRU_File = '~/.vim/vim_mru_files'
```

and it gives me the E482 error.  When the line is commented out, no error occurs.  I'm pretty new to Linux, so I don't know if this is a problem with this plugin or something configured wrong in the OS.  I have a similar line for Windows:

```
    let MRU_File = 'c:/users/jon.hatfield/documents/_vim_mru_files'
```

and I've never had any issues with this plugin on that OS.

Handy plugin, btw.  

Thanks, Jon
johatfie@gmail.com

**Comment by yegappan** [comment]
I looks like the ~/.vim directory is not present in your system. Can you try manually creating the ~/.vim directory using "mkdir ~/.vim"?

**Comment by johatfie** [comment]
The folder is already there and it does the same thing in vim and gvim.

**Comment by nelsyeung** [comment]
Pull request #11 has this fix, I hope it will be merged soon. For now you can use the actual full path, i.e., /User/username/.vim/vim_mru_files

**Comment by liuyuqiang** [comment]
using  `  let MRU_File =  $HOME . '/.vim/.vim_mru_files'  ` is successfull ,
but  `  let MRU_File =  '~/.vim/.vim_mru_files'  ` is failed .

**Comment by yegappan** [comment]
This issue is addressed by the changes made for ae72775b333cb4143a6360b67d60871e91b5f8a7.
